src/integrationTest/java/uk/gov/hmcts/reform/translate/controllers/DictionaryControllerIT.java

```diff
@@ -26,18 +31,18 @@ public class DictionaryControllerIT extends BaseTest {
     @Autowired
     private MockMvc mockMvc;
 
-    private static final String URL = "/dictionary";
-
     @Nested
     @DisplayName("Get Dictionary")
     class GetDictionary {
+        private static final String DICTIONARY_URL = "/dictionary";
```

Comment:
Is it worth having constants lile this in the controller for use accross both code, swagger and tests?

Reply:
possibly

Reply:
Resolved in #42